### PR TITLE
[Darkside] ReadMore CSS update

### PR DESCRIPTION
**Pull request by KenAJoh**

### CODE REVIEW

File: @navikt/core/react/src/provider/theme/AkselTheme.tsx [hunk 1]
```diff
@@ -3,7 +3,17 @@ import React, { forwardRef } from "react";
 import { createContext } from "../../util/create-context";
 
 type AkselThemeContext = {
-  theme: "light" | "dark";
+  /**
+   * Available color-themes
```
Comment (HalvorHaugan) [inline @navikt/core/react/src/provider/theme/AkselTheme.tsx]:
```suggestion
   * Color theme
```
```suggestion
   * Color-theme
```

File: @navikt/core/react/src/provider/theme/AkselTheme.tsx [hunk 2]
```diff
@@ -3,7 +3,17 @@ import React, { forwardRef } from "react";
...
+   * @default "light"
+   */
+  theme?: "light" | "dark";
+  /**
+   * Brand volume
+   * @default "low"
+   * This is experimental and subject name changes
```
Comment (HalvorHaugan) [inline @navikt/core/react/src/provider/theme/AkselTheme.tsx]:
```suggestion
   * This is experimental and subject to name changes
```
or just
```suggestion
   * This is experimental and subject to changes
```

File: @navikt/core/react/src/provider/theme/AkselTheme.tsx [hunk 3]
```diff
@@ -36,11 +47,12 @@ const AkselTheme = forwardRef<HTMLDivElement, AkselThemeProps>(
       hasBackgroundProp ?? (isRoot || props.theme !== undefined);
 
     return (
-      <ThemeProvider theme={theme}>
+      <ThemeProvider theme={theme} volume={volume}>
         <div
           ref={ref}
           className={cl("navds-theme", className, theme)}
           data-background={hasBackground}
+          data-volume={volume}
```
Comment (HalvorHaugan) [inline @navikt/core/react/src/provider/theme/AkselTheme.tsx:61]: Is this needed?

Comment (KenAJoh) [reply @navikt/core/react/src/provider/theme/AkselTheme.tsx:61]: In the current implementation no, but it's nice to see the current state in the DOM when testing.

**Review by HalvorHaugan — APPROVED** [review]

**Pull request merged**